### PR TITLE
Astro 6/20 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/resource-reference-hosted.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ The following table lists all available worker types on Astro Deployments.
 | A5          | 1    | 2G     |
 | A10         | 2    | 4G     |
 | A20         | 4    | 8G     |
+| A20         | 8    | 16G     |
```
Comment (chomipi88) [inline astro/resource-reference-hosted.md:54]: I believe this should be A40

@cmarteepants

**Comment by chomipi88** [comment]
Add https://github.com/astronomer/astro/issues/14074

File: astro/resource-reference-hosted.md (hunk 1 above)
Comment (jwitz) [inline astro/resource-reference-hosted.md:54]:
```suggestion
| A40         | 8    | 16G     |
```

File: astro/create-first-dag.md [hunk 2]
```diff
@@ -101,7 +101,16 @@ DAG-only deploys are an Astro feature that you can use to quickly update your As
...
 
-2. Run the following command to deploy your DAGs to Astro: 
+2. Run the following command to enable DAG-only code deploys on your Deployment.
+   
+    ```sh
+    astro deployment update --dag-deploy enable
+    ```
+   
+    When the prompt appears in the Astro CLI, select the Deployment where you want to deploy your DAGs. 
+
```
Comment (jwitz) [inline astro/create-first-dag.md:111]:
```suggestion
```

File: astro/resource-reference-hosted.md [hunk 3]
```diff
@@ -51,6 +51,8 @@ The following table lists all available worker types on Astro Deployments.
 | A5          | 1    | 2G     |
 | A10         | 2    | 4G     |
 | A20         | 4    | 8G     |
+| A40         | 8    | 16G     |
+
```
Comment (jwitz) [inline astro/resource-reference-hosted.md:55]:
```suggestion
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -27,6 +27,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- Make sure that all links to internal docs are the complete URLS including HTTPS. Otherwise the links will break in RSS. -->
 
+## June 20, 2023
+
+### Additional improvements
+
+- You can now configure Deployments with the `A40` machine type, which has 8 vCPU and 16 GiB. See [Astro hosted resource reference](resource-reference-hosted.md).
```
Comment (jwitz) [inline astro/release-notes.md:34]:
```suggestion
- You can now add new Astro users to Workspaces before the user has accepted their invite.
- You can now configure Deployments with the `A40` machine type, which has 8 vCPU and 16 GiB. See [Astro hosted resource reference](resource-reference-hosted.md).
```

**Pull request merged**